### ISSUE TITLE
I can not download from the MSStore

### ISSUE THREAD

**Issue by ViktorRezonv** [zoeyfyi/TeX-Match#121]
I find I can't download from the MSStore , they say your area doesn't support purchase(China mainland). Could you please give me another way to install. Your work is pretty good. Thank you !

**Comment by zoeyfyi** [comment]
Yes, should be relatively easy to add a `.msixbundle` build to github actions. I will look into this, thanks for the suggestion!
